modules/addons/realtimeregister_ssl/eServices/provisioning/SSLUtils.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace AddonModule\RealtimeRegisterSsl\eServices\provisioning;
+
+class SSLUtils
```

Comment:
This could also be a trait, so we don't need the static functions?

Reply:
good one